modules/configuration/jsx/DiagnosisEvolution.js

```diff
@@ -0,0 +1,386 @@
+                                btnLabel='Add Field'
+                                pendingValKey='pendingSourceField'
+                                onUserAdd={this.addSourceField}
+                                onUserRemove={this.removeSourceField}
+                            />
+                            <NumericElement
+                                name='orderNumber'
+                                min={1}
+                                max={100}
+                                label='Order Number'
+                                onUserInput={this.setFormData}
+                                value={trajectoryData.orderNumber}
+                                required={true}
+                            />
+                            <div className='btn-container'>
+                                <ButtonElement
+                                    name='submit'
+                                    label='Save'
+                                    type='submit'
+                                    onUserInput={this.handleSubmit}
+                                />
+                                <ButtonElement
+                                    label='Reset'
+                                    type='reset'
+                                    onUserInput={this.handleReset}
+                                />
+                            </div>
+                        </FieldsetElement>
+                    </FormElement>
+                </div>
+            </TabPane>
+        );
+    }
+
+    /**
+     * Renders the React component.
+     *
+     * @return {JSX} - React markup for the component
+     */
+    render() {
+        if (this.state.error) {
+            return <h3>An error occured while loading the page.</h3>;
+        }
+
+        if (!this.state.isLoaded) {
+            return <Loader />;
+        }
+
+        let tabList = [];
+        tabList.push({id: 'new', label: 'New Diagnosis Trajectory'});
+
+        let diagnosisTracks = [];
+        const trajectories = this.state.formData.diagnosisTracks;
+        if (trajectories) {
+            Object.values(trajectories).map((trajectory) => {
+                const dxID = trajectory.DxEvolutionID;
+                const dxName = trajectory.Name;
+                diagnosisTracks.push(this.renderDiagnosisForm(dxID));
+                tabList.push({id: `${dxID}`, label: dxName});
+            });
+        }
+
+        return (
+            <div>
+                <p>
+                    Use this page to manage the configuration of the study's
+                    diagnosis trajectory.
+                </p>
+                <p>
+                    To configure study subprojects
+                    <a href="{$baseurl}/configuration/subproject/"> click here
+                    </a>.
+                    To configure study projects
+                    <a href="{$baseurl}/configuration/project/"> click here</a>.
+                </p>
```

Comment:
```suggestion

```
i dont think these links are needed on this page, they also don't work for me

Reply:
yeahh i think it's fine if theyre not there, users can use the breadcrumbs to navigate to the previous page where these links can be found